assets/js/map.js

```diff
@@ -0,0 +1,15 @@
+
+var mapOptions = {
+    center: [17.385044, 78.486671],
+    zoom: 10
+}
+
+// instantiating the map with user defined mapOptions passed as parameter
+var map = new L.map("map", mapOptions);
+
+// instantiating the TileLayer Class which displays the desired map UI (passed as a String parameter)
+var layer = new L.tileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png');
```

Comment:
Looks like another `http://` slipped in here.

It is causing these warnings whenever you look around on the map: 
![image](https://user-images.githubusercontent.com/10343470/141223883-e1de8c23-a3ac-4054-bc6c-8f3eaf796c50.png)

Reply:
Ahh okay. I'll see to it.